### DISCUSSION TITLE
Aggressive global bundle adjustment

### DISCUSSION THREAD

**Discussion by youknowimcomingwhenyouhearmehumming** [stella-cv/stella_vslam#409] [General]
### How to implement an aggressive global bundle adjuster?

In my project I don't have any real time constraints. Therefore, I want to run an aggressive global bundle adjuster in the last part of my system.

I have tried to use code from `global_bundle_adjuster.cc` but this implementation only makes minor changes. It's also too fast compared to what I would expect. The result is not better even when the number of iterations is increased.

**Comment by youknowimcomingwhenyouhearmehumming** [comment]
I imagine this would have a large effect since global bundle adjustment have access to all information about the trajectory in contrast to local bundle adjustment.

**Comment by ymd-stella** [comment]
If a loop edge is not inserted, it should be.
